source/javascripts/components/PipelinesPage/hooks/usePipelineSelector.ts

```diff
@@ -1,13 +1,14 @@
 import { useCallback } from 'react';
 import { useShallow } from 'zustand/react/shallow';
+import { Pipeline } from '../../../models/BitriseYml';
 import useSearchParams from './useSearchParams';
 import useBitriseYmlStore from './useBitriseYmlStore';
 
 const usePipelineSelector = () => {
   const options = useBitriseYmlStore(
     useShallow(({ yml }) => {
       return Object.fromEntries(
-        Object.entries(yml.pipelines ?? {}).map(([pipelineKey, pipeline]) => {
+        Object.entries<Pipeline>(yml.pipelines ?? {}).map(([pipelineKey, pipeline]) => {
```

Comment:
Is the `Pipeline` type necessary here? Can't TS infer the type without it?

Reply:
Yes, TS was complaining about it.